src/test/java/org/folio/inventory/validation/experimental/InProcessNonRequestableItemStatusValidatorTest.java

```diff
@@ -0,0 +1,89 @@
+package org.folio.inventory.validation.experimental;
+
+import junitparams.JUnitParamsRunner;
+import junitparams.Parameters;
+import lombok.SneakyThrows;
+import org.folio.inventory.domain.items.Item;
+import org.folio.inventory.domain.items.ItemStatusName;
+import org.folio.inventory.domain.items.Status;
+import org.folio.inventory.exceptions.UnprocessableEntityException;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.TimeoutException;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.folio.inventory.domain.items.ItemStatusName.IN_PROCESS_NON_REQUESTABLE;
+
+@RunWith(JUnitParamsRunner.class)
+public class InProcessNonRequestableItemStatusValidatorTest {
+  private static TargetItemStatusValidators validators;
+
+  @BeforeClass
+  public static void setUp() throws Exception {
+    validators = new TargetItemStatusValidators();
+  }
+
+
+  @SneakyThrows
+  @Parameters({
+    "Available",
+    "Awaiting delivery",
+    "Awaiting pickup",
+    "In transit",
+    "Lost and paid",
+    "Missing",
+    "Order closed",
+    "Paged",
+    "Withdrawn"
+  })
+  @Test
+  public void itemCanBeMarkedAsInProcessNonRequestableWhenInAcceptableSourceStatus(String sourceStatus) {
```

Comment:
These tests are about the `non requestable` item status, please change the names accordingly. Please check the other test classes as well.

```suggestion
  public void itemCanBeMarkedAsNonRequestableWhenInAcceptableSourceStatus(String sourceStatus) {
```